src/utils/connect/fetch.ts

```diff
@@ -0,0 +1,10 @@
+import { LibraryError } from '../errors';
+
+export default (typeof fetch !== 'undefined' && fetch) ||
+  (typeof window !== 'undefined' && window.fetch) ||
+  (typeof global !== 'undefined' && global.fetch) ||
```

Comment:
```suggestion
  (typeof globalThis.fetch !== 'undefined' && globalThis.fetch) ||
```

Otherwise a discrete "global object" check per environment needs to be added. `globalThis` is the more modern cross-environment option.